digiwf-integrations/digiwf-ok.ewo-integration/digiwf-okewo-integration-core/src/main/java/de/muenchen/oss/digiwf/okewo/integration/service/PropertiesService.java

```diff
@@ -0,0 +1,19 @@
+package de.muenchen.oss.digiwf.okewo.integration.service;
+
+import de.muenchen.oss.digiwf.okewo.integration.gen.model.BenutzerType;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
```

Comment:
Könntest du hier den Constructor händisch schreiben, dann verschwindet in Idea die Fehlermeldung, dass Spring die Bean nicht finden kann.